mac

```diff
@@ -137,36 +146,18 @@ brew "pipenv"
 brew "tfenv"
 
 # Databases
-brew "postgres", restart_service: :changed
+brew "postgresql@14", restart_service: :changed
```

Comment:
I think this is why my local DB exploded a couple months back. Upstream now does the right thing so this brings us in line.